src/Components/MissionsScreen.js

```diff
@@ -1,95 +1,136 @@
-import { GET_MISSIONS } from '../gqlQueries.js';
+import { GET_MISSIONS, GET_ALL_MISSION_PROGRESS } from '../gqlQueries.js';
 import { useQuery } from '@apollo/client';
 import { useState } from 'react';
 import { useHistory } from 'react-router';
-import Button from '@material-ui/core/Button';
 import './MissionsScreen.css';
-import EcoIcon from '@material-ui/icons/Eco';
-import LinearScaleIcon from '@material-ui/icons/LinearScale';
 import HdrWeakIcon from '@material-ui/icons/HdrWeak';
 import ProgressBar from './ProgressBar.js';
 
 const MISSION_COMPONENT_PATH = "/mission";
 
+//this component displays all of the Missions in a course
 export default function MissionsScreen() {
 
+    const hist = useHistory();
+
+    const [focusedMission, setFocusedMission] = useState(null);
+
     const { loading, error, data, refetch} = useQuery(GET_MISSIONS, {
         variables: { id: "Integrated Science" },
     });
-    const hist = useHistory();
-    const [focusedMission, setFocusedMission] = useState(null);
 
-    function displayMissionList(){
-        console.log("data");
-        console.log(data);
-        return data.missions.map((mission) => (
-        <div className="Mission" onClick={()=>setFocusedMission(mission)}>
-          <HdrWeakIcon style={{ color: "white", transform: "scale(6)" }}/>
-          {/* <h1 style={{size: "10px", color: "white"}}>{mission.name}</h1> */}
-        </div>));
+    const { loading: progressLoading, error: progressError, data: progressData, refetch : progressRefetch} = useQuery(GET_ALL_MISSION_PROGRESS, {
+      variables: { id: " " },
+    });
+
```

Comment:
Great job hooking up to the backend, hopefully this API is fully implemented soon